cheatsheets/Docker_Security_Cheat_Sheet.md

```diff
@@ -224,11 +224,39 @@ To detect containers with known vulnerabilities - scan images using static analy
   - [JFrog XRay](https://jfrog.com/xray/) 
   - [Qualys](https://www.qualys.com/apps/container-security/)
 
-To detect missconfigurations in Kubernetes:
+To detect misconfigurations in Kubernetes:
  - [kubeaudit](https://github.com/Shopify/kubeaudit)
  - [kubesec.io](https://kubesec.io/)
  - [kube-bench](https://github.com/aquasecurity/kube-bench)
 
+To detect misconfigurations in Docker:
+ - [inspec.io](https://www.inspec.io/docs/reference/resources/docker/)
+ - [dev-sec.io](https://dev-sec.io/baselines/docker/)
+
+## RULE \#10 - Set the logging level to at least INFO
+
+Set Docker daemon log level to 'info'. Rationale: Setting up an appropriate log level, configures the Docker daemon to log events that you would want to review later. A ase log level of 'info' and above would capture all logs except debug logs. Until and unless required, you should not run docker daemon at 'debug' log level.
+
+To configure the log level per container:
+
+```
+$ docker --log-level info run -it alpine
```

Comment:
Following the [Docker CLI documentation](https://docs.docker.com/engine/reference/commandline/cli/), by default, the log level is info. I am with the rule as is, but this will create confusion and if people aren't careful would simply be making their scripts longer.

Reply:
What do you recommend, should I remove the part which shows the `--log-level` snippet?

Reply:
I would recommend remove lines 240 to 244 and telling the reader that by default, the containers run with that log level. This says that the user will search and understand that they themselves will have to go outside rule #10. What do you think?

Reply:
Yeah sounds reasonable to me :)

Reply:
Updated in commit [a33b02e04734f11c28556f1eee77f3e30a8eee8b](https://github.com/Paradoxis/CheatSheetSeries/commit/a33b02e04734f11c28556f1eee77f3e30a8eee8b)